jump-start-tutorial/exercise-6_using-routeparams.md

```diff
@@ -50,7 +50,7 @@ We need several things to get there:
   </div>
   ```
 3. Add a new route to `APPROUTES` with the path `contacts/:id` that points to `ContactsDetailComponent`
-4. Change the `<mat-item>` in `ContactsListComponent`'s view to `<a mat-item>` and use the `RouterLink` directive with a DSL configuration that points to `ContactsDetailComponent`
+4. Change the `<mat-item>` in `ContactsListComponent`'s view to `<a mat-list-item>` and use the `RouterLink` directive with a DSL configuration that points to `ContactsDetailComponent`
```

Comment:
Thanks @JWO719 ! The first one needs to be `mat-list-item` as well, would you mind updating your PR?

Reply:
didn't get exactly what you mean?

Reply:
> Change the `<mat-item>`

should be

> Change the `<mat-list-item>`

Reply:
but the sentence is correctly? I mean it just says you should change the <mat-item> like it is in the current state of development to...

I'm a bit confused to be honest....